src/services/__tests__/classService.test.ts

```
@@ -0,0 +1,629 @@
+      select: jest.fn().mockReturnThis(),
+      eq: jest.fn().mockReturnThis(),
+      or: jest.fn().mockReturnThis(),
+      not: jest.fn().mockReturnThis(),
+      ilike: jest.fn().mockReturnThis(),
+      order: jest.fn().mockReturnThis(),
+      range: jest.fn().mockReturnThis(),
+      delete: jest.fn().mockReturnThis(),
+      insert: jest.fn().mockReturnThis(),
+      update: jest.fn().mockReturnThis(),
+      single: jest.fn().mockResolvedValue(finalResponse || { data: null, error: null }),
+      then: jest.fn().mockResolvedValue(finalResponse || { data: [], error: null })
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)